// TRY/CATCH

/* const name = "Jose"; */

// TRY ES EL INTENTO DE EJECUTAR ALGO Y CATCH ES LA RED DE SEGURIDAD POR SI ESO NO SE PUEDE EJECUTAR QUE TE EJECUTE EN SU LUGAR OTRA COSA, CON LA POSIBILIDAD DE IMPRIMIR EL ERROR NATIVO QUE LE VIENE POR PARAMETRO AL CATCH
try {
  console.log(name);
} catch (error) {
  console.log("La variable name no existe, por favor definela");
  console.log(error);
}

try {
    
} catch (error) {
    
}